site2/docs/functions-develop-api.md

```diff
@@ -102,6 +102,39 @@ public class ExclamationFunction implements Function<String, String> {
 
 For more details, see [code example](https://github.com/apache/pulsar/blob/master/pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/ExclamationFunction.java).
 
+The return type of the `Function` can be wrapped in a Pulsar `Record` generic which gives more control on the output message (topic, schema, properties, ...)
+You can use the `Context::newOutputRecordBuilder` method to build this `Record` output.
```

Comment:
I would say you 'must' and not you 'can'. (Even if we don't enforce that in the code)
This is stronger and we really don't want users to build their own implementations of Record.

Reply:
I changed `You can use` to just `Use`.